packages/playground/remote/src/lib/offline-mode-cache.ts

```diff
@@ -53,12 +54,16 @@ export class OfflineModeCache {
 			return;
 		}
 
-		// Get the cache manifest and add all the files to the cache
-		const manifestResponse = await fetch(
-			'/assets-required-for-offline-mode.json'
-		);
-		const websiteUrls = await manifestResponse.json();
-		await this.cache.addAll([...websiteUrls, ...['/']]);
+		try {
+			// Get the cache manifest and add all the files to the cache
+			const manifestResponse = await fetch(
+				'/assets-required-for-offline-mode.json'
+			);
+			const websiteUrls = await manifestResponse.json();
+			await this.cache.addAll([...websiteUrls, ...['/']]);
+		} catch (error) {
+			logger.warn('Error caching offline mode assets:', error);
```

Comment:
I understand this might be a dev thing to get the tests going, but why would this fail even in the test environment? Could that be why the CI fails?

Reply:
I removed this check. Assets don't exist in the local environment and this code won't run then with the new check.